job-server/src/spark.jobserver/JobInfoActor.scala

```diff
@@ -11,7 +11,7 @@ import spark.jobserver.io.{JobDAO, JobInfo}
 
 object JobInfoActor {
   // Requests
-  case class GetJobStatuses(limit: Option[Int])
+  case class GetJobStatuses(limit: Option[Int],status: Option[String])
```

Comment:
<check level="error" class="org.scalastyle.scalariform.EnsureSingleSpaceAfterTokenChecker" enabled="true">
    <parameters>
     <parameter name="tokens">ARROW, EQUALS, COMMA, COLON, IF, ELSE, DO, WHILE, FOR, MATCH, TRY, CATCH, FINALLY, LARROW, RARROW</parameter>
    </parameters>
  </check>